tests/dt/test-fillna.py

```diff
@@ -128,6 +183,17 @@ def test_fillna_grouped():
     assert_equals(DT_fillna, DT_ref)
 
 
+def test_fillna_grouped_value():
+    DT = dt.Frame([[15, None, 136, 93, 743, None, None, 91],
+                  ['a','a','a','b','b','c','c','c']])
+    DT_fillna = DT[:, fillna(f[:], f.C0.mean()), by(f[-1])]
+    DT_ref = dt.Frame({
+                'C1':['a','a','a','b','b','c','c','c'],
+                'C0':[15, 75.5, 136, 93, 743, 91, 91, 91]/dt.float64,
+             })
+    assert_equals(DT_fillna, DT_ref)
+
+
```

Comment:
I think we should add a test to make sure `dt.fillna(col, value=VALUE)` matches `dt.ifelse(isna(col), VALUE, col)`